webapp/src/Logger/VarargsLogMessageProcessor.php

```diff
@@ -18,14 +18,14 @@ class VarargsLogMessageProcessor implements ProcessorInterface
      */
     public function __invoke(array $record): array
     {
-        if (strpos($record['message'], '%') === false || empty($record['context'])) {
+        if (!str_contains($record['message'], '%') || empty($record['context'])) {
             return $record;
         }
 
         $res = false;
         try {
             $res = vsprintf($record['message'], $record['context']);
-        } catch (ValueError $e) {}
+        } catch (ValueError) {}
```

Comment:
phpcs (github actions) is complaining here, so we should probably change its rules

Reply:
I think its to prevent against:
`} catch (ValueError) { command }`

Would we mind such code?

Reply:
This seems to be the same 'discussion' as the empty constructor.
I don't like `{ command }` but would be fine with `{}`. Probably we can 'teach' phpcs that (and if we replace it by php-cs-fixer that one as well)

Reply:
Mmm nope, php-cs-fixer at least doesn't allow it: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/5475 
They currently link to PSR2 which doesn't allow it.

Reply:
Yeah, empty {} is fine, non-empty should be in separate lines

Reply:
I doubt we can teach the github action the difference. So we either have to take out the rule completely or accept that it will make annotations about it.

Reply:
I'm fine with taking the rule out.